### DISCUSSION TITLE
Owon THS317-ET remote temperature sensor test - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#16] [Comments]
# Owon THS317-ET remote temperature sensor test - Haade.fr

The Multi-sensor is used to measure the ambient temperature & humidity with built-in sensor and external temperature with remote probe. It is available to detect motion, vibration and allows you to receive notifications from mobile app. The above functions can be customized, please use this guide according to your customized functions.

https://haade.fr/en/blog/test-outdoor-temperature-sensor-owon-ths317-et

**Comment by ZaGaPonG** [comment]
Bonjour,

Je viens de faire l'intégration l'appairage avec Z2M.
Module reconnu et supporté par contre aucun report depuis plus de 8h...
J'ai tenté de faire réagir la sonde (changement important de température pendant plusieurs minutes ) mais cela n'a aucun effet.
Reset de la sonde et réappairage mais toujours rien..
N'ayant pas d'autre device zigbee qui ne sont pas alimenté via secteur, y a t'il une manip particulière à faire svp?

Merci d'avance.

**Reply by haade-administrator** [reply]
Salut 
- vérifie si l'entretien est sur vrai dans l'onglet à propos du module z2m
- vérifie le schéma voir si le module est bien connecté à l'un des routeurs
-  si ton réseau n'est pas constitué d'un nombre relativement important de modules alors vérifie si l'owon n'est pas trop éloigné de la clé zigbee

**Reply by ZaGaPonG** [reply]
Hello,

Malheureusement tout est bien OK.

Je pense que le problème vient du module car aucun report en 24h alors qu'il est posé juste à côté du coordinateur :/
 J'ai testé sur l'installation zigate d'un ami, même résultat, aucun report en plus de 2h de temps même en simulant de gros changements de température :(
J'espère que le SAV Domadoo est bien.

Merci pour ton retour en tout cas :)
